<launch>

    <!-- <node pkg="drone_prediction_network2" name="drone_trajectory_prediction" type="prediction_node.py" output="screen"> -->

        <!-- set parameters -->
        <param name="n_robots" value="6" />
        <param name="n_obstacles" value="0" />
        <param name="past_horizon" value="20" />
        <param name="prediction_horizon" value="20" />
        <param name="dt" value="0.05" />

    <!-- </node> -->

</launch>
